source/SH3/system/glbuffer.cpp

```diff
@@ -0,0 +1,52 @@
+/** @file
+ *
+ *  Implementation of @ref glbuffer.hpp
+ *
+ *  @copyright 2017 Palm Studios
+ *
+ *  @date 8-6-2017
+ *
+ *  @author Jesse Buhagiar
+ */
+#include "SH3/system/glbuffer.hpp"
+#include "SH3/system/log.hpp"
+
+#define GL_VBO_UNBIND 0
+
+using namespace sh3_gl;
+
+void sh3_glbuffer::Create()
+{
+    GLenum err;
```

Comment:
Leftover code I should have removed. `glGetErorr( )` is actually a VERY bad function. It returns an error from any last glXYZ call without being cleared. We could have an issue in another part of the code that would trigger the error handler to be called, even if our buffer creation was successful. I'll remove these.